pygitguardian/client.py

```diff
@@ -36,6 +35,10 @@
 MAX_TAR_CONTENT_SIZE = 30 * 1024 * 1024
 
 
+class TotalFilesSizeExceeded(Exception):
```

Comment:
Maybe `ContentTooLarge` would be a bit simpler?

Also can you:
1. Add a short docstring to explain when this can be raised
2. Add the class to `__init__.py`